src/utils/optionsHaveIgnoredPseudoClass.js

```diff
@@ -0,0 +1,17 @@
+import matchesStringOrRegExp from "./matchesStringOrRegExp"
```

Comment:
They currently have 100% coverage by proxy. 

Feel free to add the tests to this PR though.